Web/Edubase.Web.UI/Assets/Scripts/Entry/manage-academy-openings.js

```diff
@@ -383,7 +398,7 @@ const academyOpenings = new Vue({
 
       const datePages = this.openingAcademies.filter(function (opening) {
         const oDateParts = opening.openingDate.split('-');
-        const oDateMonth = parseInt(oDateParts[1], 10) -1;
+        const oDateMonth = parseInt(oDateParts[1], 10) - 1;
```

Comment:
?

Reply:
It's not really a change. It's a code format spacing.
This method was already present